vulcan/lib/client/jsx/components/workflow/session/session_manager.tsx

```diff
@@ -42,7 +64,30 @@ export default function SessionManager() {
 
   const saveSession = useCallback(() => {
     if (hasPendingEdits) {
-      if (!confirm('You have unfilled inputs and/or uncommitted changes. Unfilled inputs will be left out of the saved state. Uncommited changes will be reset to their previously committed values. Proceed?')) {
+      if (!confirm('Pending edits will be discarded when saving. Proceed?')) {
+        return;
+      }
+    }
+
+    let params = { ...session }
+
+    if (!params.title) {
+      params.title = prompt('Set a title for this figure');
+      if (!params.title) return;
+    }
+
+    if (params.figure_id) {
+      json_post(`/api/${params.project_name}/figure/${params.figure_id}/update`, params)
```

Comment:
Fairly minor nit -- is there any feedback that the update was successful or in progress?
Also, is there any risk of double clicking save and having a race condition for update?
Maybe we could track the state of saving with some state, `isSaving` to help with both?
But fairly minor.